compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -1136,6 +1136,7 @@ impl<'tcx> Stable<'tcx> for rustc_middle::mir::ConstantKind<'tcx> {
                     let const_val = tables.tcx.valtree_to_const_val((c.ty(), val));
                     stable_mir::ty::ConstantKind::Allocated(new_allocation(self, const_val, tables))
                 }
+                ty::ParamCt(param) => stable_mir::ty::ConstantKind::ParamCt(opaque(&param)),
                 _ => todo!(),
```

Comment:
Maybe make this match exhaustive and mark error/infer as unreachable and the others as unimplemented.